sdk/schemaregistry/azure-schemaregistry-avroencoder/azure/schemaregistry/encoder/avroencoder/_schema_registry_avro_encoder.py

```diff
@@ -71,42 +71,38 @@ class AvroEncoder(object):
 
     """
 
-    def __init__(self, **kwargs):
-        # type: (Any) -> None
-        try:
-            self._schema_registry_client = kwargs.pop(
-                "client"
-            )  # type: "SchemaRegistryClient"
-        except KeyError as exc:
-            raise TypeError(f"'{exc.args[0]}' is a required keyword.")
+    def __init__(
+        self,
+        client: "SchemaRegistryClient",
+        group_name: Optional[str] = None,
```

Comment:
was this intentional to change these from being kwargs ?

Reply:
yes, changing them to py 3 typing